### DISCUSSION TITLE
grayscale scans distorted, but only certain page sizes and DPI's

### DISCUSSION THREAD

**Discussion by jbbythebch** [cyanfish/naps2#213] [Support]
Scans using grayscale are only partially working, depending on the page size and DPI.

The ones that fail are distorted and skewed to the right, mostly illegible.

Page size up to A4 works fine, up to 300 DPI.  Larger page size, even US Letter, are illegible.  DPI settings larger than 300 are also illegible on A4.

For US Letter size, no DPI works, not even 100.

I looked at the NAPS2 error logs, and found nothing.

This seems similar to part of ticket #1029, also running on macOS Ventura, on SourceForge:  https://sourceforge.net/p/naps2/tickets/1029/
(But, I posted this here due to something I read saying GitHub is the future for NAPS2.)

On that SourceForge post, the submitter described similar results as zebra stripes, which is a pretty good description.  His 'test.pdf' image is showing the same thing that my unsuccessful scans are showing.

Here's what I'm getting too:
[failed grayscale scan, 100dpi, US Letter size.pdf](https://github.com/cyanfish/naps2/files/13095848/failed.grayscale.scan.100dpi.US.Letter.size.pdf)

Glass or Feeder has no effect. 
Image quality of 10 or 75, or somewhere in between, no effect.
Only the page size seems to cause the issue.

Running on:
- Brother MFC-J880DW (multifunction, fairly inexpensive printer/scanner)
- MacBook Pro, Intel, Ventura 13.6
- and on MacBook Pro, Apple Silicon, Ventura 13.6 (same issue)
- NAPS2 version 7.1.2.0

I have sort of a workaround of dropping the hue and saturation to -1000 and then save the page(s), and this seems like it's grayscale, but I didn't spend much time looking at these edits vs a successful grayscale.

Using the built-in macOS scanning software, or ABBYY FineReader Pro, grayscale works fine.
(By the way, I really like this great NAPS2 software much better than the ABBYY software, thanks!!)

Any tests I can try, or any more info requested, or a better workaround or fix, let me know.

Thanks!

**Comment by cyanfish** [comment]
Can you follow the [instructions here](https://www.naps2.com/doc/troubleshooting#debug-log) to collect debug logs?

**Reply by jbbythebch** [reply]
Wow, you're fast.  Thanks.

Here's the debug log:
[debuglog.txt](https://github.com/cyanfish/naps2/files/13110861/debuglog.txt)

Same result as above for the scan looking like zebra stripes.

**Comment by cyanfish** [comment]
The grayscale issue should be fixed in [NAPS2 7.2.1](https://www.naps2.com/download).

**Comment by jbbythebch** [comment]
Thanks!!

Confirmed fixed - grayscale works perfectly now.
